src/service/util/team.ts

```diff
@@ -26,3 +27,13 @@ export function teamFromControl(control: number): number {
         return 0;
     }
 }
+
+export function isRoundOver(world: CountryData[]): boolean {
+    const axisSpawns = world.some(
+        (c) => c.isActiveSpawn && c.isSpawn && c.initialControl === 0
+    );
+    const alliesSpawns = world.some(
+        (c) => c.isActiveSpawn && c.isSpawn && c.initialControl === 100
+    );
+    return alliesSpawns || axisSpawns;
```

Comment:
yes, but now it needs to be inverted >.>
`!alliesSpawns || !axisSpawn`

Reply:
actually, why check for both `isSpawn` and `isActiveSpawn`? Isn't that redundant?